functions/validator/utils.py

```diff
@@ -0,0 +1,30 @@
+__version__ = '0.0.1'
+
+import os
+import logging
+import time
+from datetime import datetime
+
+
+def set_stream_logger(name="cis-validator", level=logging.INFO,
+                      format_string=None):
+    """
+    """
+
+    if format_string is None:
```

Comment:
Let's use ``if not format_string`` because:

* It's more pythonic
* This doesn't match an empty string but only ``NoneType``